README.md

```diff
@@ -1,8 +1,17 @@
-# Toit Sparkfun Quiic Motor Driver package 
+# Toit SparkFun Quiic Motor Driver
+
+Driver for the SparkFun Qwiic motor driver.
```

Comment:
The first paragraph of the README is used as description of the package. (Unless the package.yaml file has a "description" entry..